src/session/client-certificate.c

```diff
@@ -285,7 +285,7 @@ sssd_map_certificate (const char *certificate, char** username)
  *
  * Read the given certificate file, ensure that it belongs to our own cgroup, and ask
  * sssd to map it to a user. If everything matches as expected, return the user name.
- * Otherwise return %NULL, a warning message will already have been logged.
+ * Otherwise exit the process with sending an appropriate error to the protocol.
```

Comment:
"to the protocol" is a bit weird as language.  "To stdout using the cockpit protocol" perhaps?

Reply:
Fixed, thanks!